### ISSUE TITLE
NRE in Tools.HandleHotkeys()

### ISSUE THREAD

**Issue by NoirFry** [pardeike/CameraPlus#12]
NRE here:
```c#
var map = Find.CurrentMap;
var savedViews = map.GetComponent<SavedViews>();
```
as map can be `null` when you have a caravan and abandon your last settlement.
In this case pressing any speed change button (1, 2, 3) will result in NRE.
I added null check here and it's working fine to me.
```c#
var map = Find.CurrentMap;
if (map == null)
    return;
```

**Comment by pardeike** [comment]
Thanks for this specific report. Will be fixed in next version.

**Comment by pardeike** [comment]
Fixed in upcoming v2.0.2
